src/um/audiopolicy.rs

```diff
@@ -0,0 +1,173 @@
+    NewNotifications: *mut IAudioSessionEvents,
+  ) -> HRESULT,
+  fn UnregisterAudioSessionNotification(
+    NewNotifications: *mut IAudioSessionEvents,
+  ) -> HRESULT,
+}}
+RIDL!{#[uuid(0xbfb7ff88, 0x7239, 0x4fc9, 0x8f, 0xa2, 0x07, 0xc9, 0x50, 0xbe, 0x9c, 0x6d)]
+interface IAudioSessionControl2(IAudioSessionControl2Vtbl):
+  IAudioSessionControl(IAudioSessionControlVtbl) {
+  fn GetSessionIdentifier(
+    pRetVal: *mut LPWSTR,
+  ) -> HRESULT,
+  fn GetSessionInstanceIdentifier(
+    pRetVal: *mut LPWSTR,
+  ) -> HRESULT,
+  fn GetProcessId(
+    pRetVal: *mut DWORD,
+  ) -> HRESULT,
+  fn IsSystemSoundsSession() -> HRESULT,
+  fn SetDuckingPreference(
+    optOut: BOOL,
+  ) -> HRESULT,
+}}
+RIDL!{#[uuid(0xbfa971f1, 0x4d5e, 0x40bb, 0x93, 0x5e, 0x96, 0x70, 0x39, 0xbf, 0xbe, 0xe4)]
+interface IAudioSessionManager(IAudioSessionManagerVtbl): IUnknown(IUnknownVtbl) {
+  fn GetAudioSessionControl(
+    AudioSessionGuid: LPCGUID,
+    StreamFlags: DWORD,
+    SessionControl: *mut *mut IAudioSessionControl,
+  ) -> HRESULT,
+  fn GetSimpleAudioVolume(
+    AudioSessionGuid: LPCGUID,
+    StreamFlags: DWORD,
+    AudioVolume: *mut *mut ISimpleAudioVolume,
+  ) -> HRESULT,
+}}
+RIDL!{#[uuid(0xc3b284d4, 0x6d39, 0x4359, 0xb3, 0xcf, 0xb5, 0x6d, 0xdb, 0x3b, 0xb3, 0x9c)]
+interface IAudioVolumeDuckNotification(IAudioVolumeDuckNotificationVtbl): IUnknown(IUnknownVtbl) {
+  fn OnVolumeDuckNotification(
+    sessionID: LPCWSTR,
+    countCommunicationSessions: UINT32,
+  ) -> HRESULT,
+  fn OnVolumeUnduckNotification(
+    sessionID: LPCWSTR,
+  ) -> HRESULT,
+}}
+RIDL!{#[uuid(0x641dd20b, 0x4d41, 0x49cc, 0xab, 0xa3, 0x17, 0x4b, 0x94, 0x77, 0xbb, 0x08)]
+interface IAudioSessionNotification(IAudioSessionNotificationVtbl): IUnknown(IUnknownVtbl) {
+  fn OnSessionCreated(
+    NewSession: *mut IAudioSessionControl,
+  ) -> HRESULT,
+}}
+RIDL!{#[uuid(0xe2f5bb11, 0x0570, 0x40ca, 0xac, 0xdd, 0x3a, 0xa0, 0x12, 0x77, 0xde, 0xe8)]
+interface IAudioSessionEnumerator(IAudioSessionEnumeratorVtbl): IUnknown(IUnknownVtbl) {
+  fn GetCount(
+    SessionCount: *mut c_int,
+  ) -> HRESULT,
+  fn GetSession(
+    SessionCount: c_int,
+    Session: *mut *mut IAudioSessionControl,
+  ) -> HRESULT,
+}}
+RIDL!{#[uuid(0x77aa99a0, 0x1bd6, 0x484f, 0x8b, 0xc7, 0x2c, 0x65, 0x4c, 0x9a, 0x9b, 0x6f)]
+interface IAudioSessionManager2(IAudioSessionManager2Vtbl):
+  IAudioSessionManager(IAudioSessionManagerVtbl) {
+  fn GetSessionEnumerator(
+    SessionEnum: *mut *mut IAudioSessionEnumerator,
+  ) -> HRESULT,
+  fn RegisterSessionNotification(
+    SessionNotification: *mut IAudioSessionNotification,
+  ) -> HRESULT,
+  fn UnregisterSessionNotification(
+    SessionNotification: *mut IAudioSessionNotification,
+  ) -> HRESULT,
+  fn RegisterDuckNotification(
+    sessionID: LPCWSTR,
+    duckNotification: *mut IAudioVolumeDuckNotification,
+  ) -> HRESULT,
+  fn UnregisterDuckNotification(
+    duckNotification: *mut IAudioVolumeDuckNotification,
+  ) -> HRESULT,
+}}
```

Comment:
This file has incorrect indentation. It should be 4 space indentation.

Reply:
sorry, fixed